src/classlibnative/inc/nlsinfo.h

```diff
@@ -101,10 +101,8 @@ class COMNlsInfo {
     //
     static FCDECL1(FC_BOOL_RET, nativeInitCultureData, CultureDataBaseObject *data);
     static FCDECL3(FC_BOOL_RET, nativeGetNumberFormatInfoValues, StringObject* localeNameUNSAFE, NumberFormatInfo* nfi, CLR_BOOL useUserOverride);
-#ifdef FEATURE_USE_LCID
```

Comment:
How do I rationalize this with the above change to `clr.coreclr.props`?  Are we just removing `FEATUTRE_USE_LCID` from the product?